### PR TITLE
Requirements setting #164: Kiali Operator

### PR DESCRIPTION
**Pull request by svatwork**
Issue: #164 
Task: Kiali operator added and configured

Apologies for being naive. Here, I have a basic configuration of Kiali Operator, but that's of course not sufficient. But I don't know what is under the definition of done for this story (or for any story adding new components), is it a basic configuration like this or are there any special requirements for integrating it? 

On the top of my head:

- [ ] Refactor this chunk from `values/istio-operator-raw.gotmpl`:
```
kiali:
          enabled: {{ $ia | get "kiali.enabled" true }}
          k8s:
            priorityClassName: "otomi-critical"
            resources:
              {{- with $i | get "resources.kiali" nil }}
                {{- toYaml . | nindent 14 }}
              {{- else }}
              requests:
                cpu: 100m
                memory: 512Mi
              limits:
                cpu: 500m
                memory: 1
                {{- end }}
      values:
        kiali:
          contextPath: '/kiali'
          dashboard:
            auth:
              strategy: anonymous #openid
              # openid:
              #   issuer_uri: {{ $keycloakBase }}
              #   client_id: {{ $v.oidc.clientID }}
            jaegerURL: https://{{ $appsDomain }}/tracing/
            grafanaURL: https://{{ $appsDomain }}/grafana-istio/
```
- [ ] Set new Istio version (breaking change), maybe work on #259 first.
- [ ] Configure Kiali CR (https://kiali.io/documentation/latest/installation-guide/#_create_or_edit_the_kiali_cr)
- [ ] Expose `values.yaml` in `values-schema.yaml` (and then we should make a distinction between what should be default and what should be configurable).

### CODE REVIEW

File: charts/istio-operator/Chart.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-version: 1.5
+version: 1.9-dev
 tillerVersion: ">=2.7.2"
 description: Helm chart for deploying Istio operator
 keywords:
   - istio
   - operator
 sources:
-  - http://github.com/istio/istio/operator
+  - https://github.com/istio/istio/operator
```
Comment (Morriz) [inline charts/istio-operator/Chart.yaml:10]: this is actually not the correct location as that is deprecated

File: charts/istio-operator/Chart.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 apiVersion: v1
 name: istio-operator
-version: 1.5
+version: 1.9-dev
```
Comment (Morriz) [inline charts/istio-operator/Chart.yaml:3]: can we target stable? so 1.8 probably

**Review by Morriz — CHANGES_REQUESTED** [review]
seems like a work in progress

**Comment by svatwork** [comment]
> seems like a work in progress

Definitely, but I really wanted to know if I'm heading in the right direction. Would you please take a look at the requirements I made and tell me if I'm in the right direction? @Morriz

**Comment by j-zimnowoda** [comment]
@svatwork making things clear is not naive IMO.
Thanks for making it more precise. Just maybe it is better to elaborate an issue itself.

For the code indeed work in progress. I guess you need some artefacts so Kiali operator knows what to do.

**Pull request closed without merging**